hooks/use-curriculums.js

```diff
@@ -0,0 +1,17 @@
+import useFetch from "./use-fetch";
+
+const useCurriculums = () => {
+  const { data, error } = useFetch(`/api/curriculums`);
+
+  if (error || !data) return undefined;
+  const curriculums = data
+    .map((curriculum) => ({
+      text: curriculum.name,
+      url: `/curriculums/${curriculum.id}`,
+    }))
+    .slice(0, 3);
```

Comment:
I think that .slice(0, 3); it is too obscure. Could be better to define a constant with a name.  Like "numberOfCurriculums"?

Reply:
Agree, done